Add-ons/UmbracoForms/Developer/Forms-in-the-Database/index.md

```diff
@@ -8,6 +8,8 @@ As of Umbraco Forms version 8.5.0 it is possible to persist all forms data in th
 
 In this article you will find instructions on how to migrate your Umbraco Forms data to the database.
 
+As of Umbraco Forms version 9.0.0 it is *only* possible to store form data in the database.  As such, if upgrading to Umbraco 9 and using Forms, you should first migrate the forms to the database using Forms 8.
```

Comment:
Does this article even make sense for Umbraco 9?
Wouldn't it be better to instead have an article (or even just a section somewhere) that explains that all forms data is stored in the database.
And then have some notes in migration/upgrading guides on how to migrate the forms to the database? 🤔

Reply:
I think this is probably OK - i.e. if you read this, and you are on V9, there's just this particular note to be aware of.  But feel free to re-work this if you prefer.